src/main/java/core/basesyntax/service/impl/ReportReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReportReaderImpl implements ReportReader {
```

Comment:
```suggestion
public class ReportReaderCsv implements ReportReader {
```
We can read from database or from command line